CHANGES

```diff
@@ -1,3 +1,4 @@
+20190827 remove maildirwatch, you can do this more efficiently using *notify() syscalls
```

Comment:
maybe just "remove inefficient maildirwatch"?

Reply:
Yes, the rest could be in the commit message.